infra/enrolment-records-store/src/main/java/com/simprints/infra/enrolment/records/store/remote/EnrolmentRecordApiInterface.kt

```diff
@@ -3,11 +3,13 @@ package com.simprints.infra.enrolment.records.store.remote
 import com.simprints.infra.enrolment.records.store.remote.models.ApiEnrolmentRecords
 import com.simprints.infra.network.SimRemoteInterface
 import retrofit2.http.Body
+import retrofit2.http.Headers
 import retrofit2.http.POST
 import retrofit2.http.Path
 
 internal interface EnrolmentRecordApiInterface : SimRemoteInterface {
 
+    @Headers("Content-Encoding: gzip")
```

Comment:
FYI we have not added support for content encoding on this endpoint on our side.
But it makes sense, I'll add it early next week.

Reply:
@ethiery wouldn't it be simple to just blank enable gzip support across endpoints based on the headers?

Reply:
It would not make much difference complexity wise on BFSID's side. 
I think the only reason I did not propose a blanket compression is that compressing small payloads is most likely counter-productive (as in, the latency added by SID compressing and BFSID decompressing will exceed the latency decrease caused by the tiny bandwidth savings). But it's hard to say for sure without actually benchmarking it.

Reply:
If it's all the same to you, I'd rather not fix anything that is not broken and keep the compression/decompression for endpoints that see large request/response bodies.

Reply:
I didn't mean it as compress everything. Just having the capability to accept compressed payloads if the `Content-Encoding` header is present. You would not need to think about it for any future changes.

On SID side that is a capability built into the OkHttp by default (for response bodies).